### DISCUSSION TITLE
Cannot get demo to run for iOS

### DISCUSSION THREAD

**Discussion by ursus-the-bear** [ionic-team/capacitor#7444] [Q&A]
I have created a totally empty Angular app using the following commands:

ng new cap-test
cd cap-test
npm i @capacitor/core
npm i -D @capacitor/cli
npx cap init
npm i @capacitor/android @capacitor/ios
npx cap add ios
npx cap sync -> got the error that index.html is missing so updated capacitor.config.ts webDir -> dist/cap-test/browser
ng build
npx cap sync

and finally

npx cap run ios

which gives me this error: xcodebuild: error: The workspace named "App" does not contain a scheme named "App". The "-list" option can be
        used to find the names of the schemes in the workspace.

which makes sense as if I do a 
     npx cap open ios
The App.xcworkspace is opened and there are no schemes defined in that config. If I use XCode to open the App.xcodeproj file I get the schemes?

I have tried adding the scheme to the workspace but that does not work, I have tried recreating everything but no matter what I do I cannot get it to work. 

I am using all the newest versions of Angular (17.3.6), XCode 15.3 and the newest capacetor version - any ideas as to how to get this to work?

I have done a npx cap doctor ios and get the success - iOS looking great!

**Comment by DepkaCZ** [comment]
Having the exactly same issue, the workspace is EMPTY and when i try cap run ios i get the same error with missing scheme :(

**Comment by DepkaCZ** [comment]
@ursus-the-bear Ok i found my missing puzzle piece :) Check this guide:

https://capacitorjs.com/docs/getting-started/environment-setup

The thing i was missing was CocoaPods. Once i have installed it everything started working like a charm :)

**Comment by ursus-the-bear** [comment]
exactly, was just about to post it here for you. the command for this problem, in case somebody does not want to read the manual: sudo gem install -n /usr/local/bin cocoapods

**Reply by janeon** [reply]
Alternatively, install with homebrew since that's also a dependency: 

`brew install cocoapods`

**Reply by jayenashar** [reply]
docs are a little conflicting here (**_emphasis_** mine):
> Homebrew is a package manager for macOS packages. You _**need**_ to install it in order to install CocoaPods for both Intel and Apple Silicon Macs.
> You can install CocoaPods directly with Ruby Gem.

**Reply by janeon** [reply]
Good point, curious if they conflict at all for purposes of installing cocoapods?

**Reply by jayenashar** [reply]
ah well i installed homebrew.  probably will need it for other things down the line
